release-notes/opensearch-security.release-notes-2.11.0.0.md

```diff
@@ -0,0 +1,40 @@
+## 2023-10-18 Version 2.11.0.0
+
+Compatible with OpenSearch 2.11.0
+
+### Enhancements
+* Authorization in Rest Layer ([#2753](https://github.com/opensearch-project/security/pull/2753))
+* Improve serialization speeds ([#2802](https://github.com/opensearch-project/security/pull/2802))
+* Integration tests framework ([#3388](https://github.com/opensearch-project/security/pull/3388))
```

Comment:
We should remove this line since the feature was pulled from the 2.11 release :'(

Reply:
Wait, the Integration tests framework? or the admin related changes? I think you are talking about admin related changes in which case you might have reviewed stale copy of this PR 😅